src/bun.js/node/node_perf_hooks_histogram_binding.zig

```diff
@@ -0,0 +1,234 @@
+const std = @import("std");
+const bun = @import("root").bun;
+const HDRHistogram = @import("hdr_histogram.zig").HDRHistogram;
+const meta = bun.meta;
+const JSC = bun.JSC;
+const JSValue = JSC.JSValue;
+
+// Wrapper around HRD Histogram
+pub const RecordableHistogram = struct {
+    pub usingnamespace JSC.Codegen.JSRecordableHistogram;
+    hdrHist: HDRHistogram = undefined,
```

Comment:
`undefined` as the default value in a field is almost always a footgun. i suggest removing the default, and then in the `bun.new` call, you can pass the histogram there.